flow-typed/redux/transactions.js

```diff
@@ -8,6 +8,7 @@ declare type TransactionFromBlockExplorer = {|
   +hash: Hash,
   +to: string,
   +from: string,
+  +input: string,
```

Comment:
I think we need to leave comment here. When this PR will be closed nobody will know what is this

Reply:
This field's name does not differ from ETH node getTransaction response format.
But, ok, will add some note in next PR